### DISCUSSION TITLE
question: Is posible to launch epoptes from a different machine than a classroom server?

### DISCUSSION THREAD

**Discussion by N0rbert** [epoptes/epoptes#101] [Q&A] [answered]
I'm running LTSP server on headless Ubuntu 16.04 LTS server with 50 fat clients and I want to use Epoptes on different machine in the same net.
How should I correctly do this?
Currently I use `ssh -X user@ltsp-server epoptes` but it is not stable. It crashes sometimes.

Do we have better method?

**Comment by alkisg** [comment, marked as answer]
Hi, this is a duplicate of #33 and #100.
